src/components/loadable-item.cjsx

```diff
@@ -18,9 +18,25 @@ module.exports = React.createClass
     renderItem: React.PropTypes.func.isRequired
     saved: React.PropTypes.func
     load: React.PropTypes.func
+    renderLoading: React.PropTypes.func
+    update: React.PropTypes.func
+
+  componentDidMount: -> @reload({})
+  componentDidUpdate: (oldProps) -> @reload(oldProps)
```

Comment:
can this be
```
  componentWillUnmount: -> @reload(@props)
```

instead?  I quickly tested through this and currently, this is causing problems for homework and navigating between problems using breadcrumbs.  really, it would seem we only need to reload on the component unmount and this worked for the use cases described in the PR for me